backends/p4tools/modules/testgen/test/gtest_utils.cpp

```diff
@@ -31,7 +31,7 @@ std::optional<const P4ToolsTestCase> P4ToolsTestCase::create(
     if (!compilerResults.has_value()) {
         return std::nullopt;
     }
-    return P4ToolsTestCase{compilerResults.value()->getProgram()};
+    return P4ToolsTestCase(compilerResults.value().get().getProgram());
```

Comment:
Since you have already checked for nullopt, you can use `compilerResults->get()` if you want (`operator->` and `operator*` are unchecked, unlike `value()`).